### ISSUE TITLE
 Compilation error in Ivy mode with 9.0.0-next.5

### ISSUE THREAD

**Issue by sergey-morenets** [angular/angular#32506]
# 🐞 bug report

### Is this a regression?

No

### Description

I have a working Angular v8 application that uses @agm/core(1.0.0-beta.7) dependency. I tried to upgrade to 9.0.0-next.5 to verify that Ivy mode works properly:

"angularCompilerOptions": {
"enableIvy": true
}

But in Ivy mode I get the error messages. In non-Ivy mode everything works fine.

## 🔬 Minimal Reproduction

ng build --prod

## 🔥 Exception or Error

```
error TS8002: 'longitude' is not a valid property of <agm-map>.
<agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom">

error TS8002: 'zoom' is not a valid property of <agm-map>.
<agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom">

error TS8001: 'agm-marker' is not a valid HTML element.
<agm-marker [latitude]="latitude" [longitude]="longitude"></agm-marker>
```

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 9.0.0-next.3
Node: 12.3.1
OS: win32 x64
Angular: 9.0.0-next.5
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-next.3
@angular-devkit/build-angular     0.900.0-next.3
@angular-devkit/build-optimizer   0.900.0-next.3
@angular-devkit/build-webpack     0.900.0-next.3
@angular-devkit/core              9.0.0-next.3
@angular-devkit/schematics        9.0.0-next.3
@angular/cdk                      8.1.4
@angular/cli                      9.0.0-next.3
@angular/material                 8.1.4
@ngtools/webpack                  9.0.0-next.3
@schematics/angular               9.0.0-next.3
@schematics/update                0.900.0-next.3
rxjs                              6.5.3
typescript                        3.5.3
webpack                           4.39.3
</code></pre>

**Comment by pshurygin** [comment]
AGM is not compatible with ivy. This has been fixedin the agm repo, but the fix has not yet been released to npm. Here is the relevant thread 
https://github.com/SebastianM/angular-google-maps/issues/1647

**Comment by sergey-morenets** [comment]
@pshurygin Thank you for the explanation.

But this thread seems to be closed(without publishing new agm version). So how should we react? Give up the idea of using Ivy in our application or move to other Google Maps library?

**Comment by pshurygin** [comment]
I guess it will be published eventually, as the fix is already merged into master branch.

**Comment by JoostK** [comment]
Looked into this just now, `@agm/core` 1.0.0 has bundles which has decorators represented in a way that is strange to me. I'm looking into it.

**Comment by JoostK** [comment]
@sergey-morenets Could you please test again with the `9.0.0-next.10` release, as I believe the issue with `@agm/core` has been resolved in #32901 (which is in `next.9`, even).

**Comment by sergey-morenets** [comment]
@JoostK Yes, it's not reproduced any more with the latest version of @agm/core
